packages/ember-htmlbars/lib/keywords/closure-component.js

```diff
@@ -74,12 +74,14 @@ export function isComponentCell(component) {
 }
 
 function createNestedClosureComponentCell(componentCell, params, hash) {
+  let newHash = assign(Object.create(null), hash);
```

Comment:
I share @rwjblue hesitations about the prototype chain being the best way to resolve this.